app/services/course/assessment/submission/statistics_download_service.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+require 'csv'
+class Course::Assessment::Submission::StatisticsDownloadService
```

Comment:
Is it possible to add a test for this service?

Reply:
OK, I will add test case as well.